src/main/java/com/opams/arch/entities/ProductEntity.java

```diff
@@ -1,8 +1,38 @@
 package com.opams.arch.entities;
 
-import javax.persistence.Entity;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.math.BigDecimal;
 
 @Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
 public class ProductEntity {
+
+    @GeneratedValue
+    @Id
+    private long productId;
```

Comment:
Maybe we should use Boxed type Long instead of plain long?
Cause we will create JPA repository in future, that resolve id type.
What do you think?